ui/appui-react/src/appui-react/syncui/SyncUiEventDispatcher.ts

```diff
@@ -59,6 +59,10 @@ export enum SyncUiEventId {
   /** The current object the reads and write UI State has changed. */
   UiStateStorageChanged = "uistatestoragechanged",
   ShowHideManagerSettingChange = "show-hide-setting-change",
+  /** The list of feature overrides applied has been changed
+   * @alpha
```

Comment:
As this have been in the product since a couple of minors, you can make them public right now in 4.0